WMF Framework/ExploreCardCollectionViewCell.swift

```diff
@@ -23,6 +23,12 @@ public class ExploreCardCollectionViewCell: CollectionViewCell, Themeable {
     
     static let overflowImage = UIImage(named: "overflow")
     
+    public var singlePixelDimension: CGFloat = 0.5
+    open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        singlePixelDimension = traitCollection.displayScale > 0 ? 1.0/traitCollection.displayScale : 0.5
```

Comment:
You should ensure `displayScale > 0`, otherwise the pixel dimension could be negative or infinity

Reply:
hmm it looks like it's doing that...

Reply:
🤦‍♂️ my bad